### DISCUSSION TITLE
Reduce pop-in of movie artwork (on side-scroll)

### DISCUSSION THREAD

**Discussion by ThijmenDam** [seerr-team/seerr#468] [Ideas]
Hi Jellyseerr community,

Have you noticed that when you side-scroll on a list of movies (e.g. on the discovery page), the artwork of the new movies pop-in? In other words, the artwork of a movie is loaded when the image container is already visible to the user, so there is a short but clearly visible transition from the "empty" gray background to the artwork when you scroll.

To me, this really affects how fluid the application feels, so I have a suggestion.

Should we perhaps preload the artworks of the next 5(?) image containers for each side-scroll container? In this way, pop-in would be greatly reduced, especially if a user scrolls with moderate speed (which I assume is usually the case).

There is of course a trade-off in terms of initial load time as more images have to be loaded when Jellyseerr is opened, but to me this seems well worth it.

Really curious how you think about this!

**Comment by fallenbagel** [comment]
This would be a good discussion for [sct/overseerr](https://github.com/sct/overseerr/discussions/new?category=feature-requests)

**Reply by ThijmenDam** [reply]
Thanks! I made a [post](https://github.com/sct/overseerr/discussions/3606) there.
